app/src/main/java/appricottsoftware/clarity/fragments/ChannelFragment.java

```diff
@@ -336,21 +314,19 @@ public void onFailure(int statusCode, Header[] headers, Throwable throwable, JSO
                 }
                 super.onFailure(statusCode, headers, throwable, errorResponse);
             }
+
             @Override
             public void onFailure(int statusCode, Header[] headers, Throwable throwable, JSONArray errorResponse) {
                 super.onFailure(statusCode, headers, throwable, errorResponse);
                 throwable.printStackTrace();
-                Log.e(TAG, "");
             }
 
             @Override
             public void onFailure(int statusCode, Header[] headers, String responseString, Throwable throwable) {
```

Comment:
We don't need a String responseString

Reply:
@mschaffrath I'm removing this block of code.